packages/cli/README.md

```diff
@@ -51,9 +51,9 @@ default: './ai_scan_cli_output'
 
 ## Batch Scan
 
--   You should provide --inputFile option, if you provide both --url and --inputFile, single url scan will be run to scan --url.
--   Summary HTML report will be generated in the output folder, previous result will be overwritten.
--   Also an error log will be generated in case if any error.
+-   Required --inputFile option with list or URLs to scan, separated by a new line.
```

Comment:
Should this be "list of URLs" instead of "or," or do we support adding a list of URLs by command line?

Reply:
No, we do not accept list from a command line, only from a file